source_config.go

```diff
@@ -0,0 +1,15 @@
+package spanner
+
+//go:generate paramgen -output=paramgen_src.go SourceConfig
+
+type SourceConfig struct {
+	// Database is the name of the database to use. A valid database name has the
+	// form projects/PROJECT_ID/instances/INSTANCE_ID/databases/DATABASE_ID
+	Database string `json:"database" validate:"required"`
+
+	// Tables represents the spanner tables to read from.
+	Tables []string `json:"tables" validate:"required"`
+
+	// Endpoint is the URL for endpoint override - testing/dry-run only
+	Endpoint string `json:"endpoint"`
```

Comment:
shouldn't this be required?

Reply:
Same as commented before, here's the [kinesis example](https://github.com/conduitio-labs/conduit-connector-kinesis/blob/194ca3afc288a196213917b42e1c54c19dedf688/common/config.go#L24-L25)